pages/linux/mdadm.md

```diff
@@ -1,27 +1,28 @@
 # mdadm
 
 > RAID management utility.
+> More information: <https://linux.die.net/man/8/mdadm>.
 
 - Create array:
 
-`mdadm --create {{/path/to/raid_device_file}} --level {{raid_level}} --raid-devices {{number_of_disks}} {{/path/to/disk_device_file}}`
+`mdadm --create {{path/to/raid_device_file}} --level {{raid_level}} --raid-devices {{number_of_disks}} {{path/to/disk_device_file}}`
```

Comment:
I'm trying to suggest better variables using the [Arch Wiki](https://wiki.archlinux.org/index.php/RAID)
```suggestion
`mdadm --create {{/dev/md/myRAID}} --level {{raid_level}} --raid-devices {{number_of_disks}} {{/dev/sdXN}}`
```